types/beautify/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for beautify 0.0
+// Project: https://github.com/gko/beautify#readme
+// Definitions by: Zhang Nan <https://github.com/anyone-developer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const c: beautify;
```

Comment:
Consider just using a `function`:
```suggestion
declare function c(content: string, options: Options): string;
```
and then deleting the `beautify` interface.

Reply:
yes. and I find a way to use export not using declare to clarify. and function name should be 'beautify' from source code sample code. user should directly use beautify().